src/core/grid_based_algorithms/lb.cpp

```diff
@@ -2208,70 +2193,38 @@ void lb_calc_n_from_rho_j_pi(const Lattice::index_t index, const double rho,
 
 /*@}*/
 
+#include <boost/range/numeric.hpp>
+
+std::array<double, 19> lb_calc_m_from_n(const std::array<double, 19> n) {
+  std::array<double, 19> m;
+  for (int i = 0; i < 19; i++) {
+    m[i] = boost::inner_product(lbmodel.e_ki[i], n, 0.0);
```

Comment:
Do we lose any efficiency by doing this, or does the `constexpr` suffice?

Reply:
WIP